tests/test_features_chords.py

```diff
@@ -0,0 +1,104 @@
+# encoding: utf-8
+# pylint: skip-file
+"""
+This file contains tests for the madmom.features.chords module.
+
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import unittest
+from . import AUDIO_PATH, ACTIVATIONS_PATH
+
+from madmom.features import Activations
+from madmom.features.chords import *
+
+
+sample_file = "%s/sample.wav" % AUDIO_PATH
+sample_cnn_act = Activations('%s/sample.cnn_chord_features.npz' %
+                             ACTIVATIONS_PATH)
+sample_deep_chroma_act = Activations('%s/sample.deep_chroma.npz' %
+                                     ACTIVATIONS_PATH)
```

Comment:
Please use `os.path.join()` as in `tests/test_utils.py` so Windows tests do not fail because of this -- they do for some other reason nonetheless...